provider/cmd/pulumi-gen-aws-native/main.go

```diff
@@ -106,7 +108,12 @@ func main() {
 				panic(fmt.Sprintf("error generating examples: %v", err))
 			}
 			providerDir := filepath.Join(".", "provider", "cmd", "pulumi-resource-aws-native")
-			writePulumiSchema(*supportedSpec, providerDir, "schema.json")
+			if err = writePulumiSchema(*supportedSpec, providerDir, "schema.json"); err != nil {
+				break
```

Comment:
Shouldn't this (and the error cases added here) notify in some when writing fails?

Reply:
It does, but the existing code pattern is just a bit weird. When it breaks the loop there's then a check for the error. I don't like the pattern, but just keeping it matching the code around it.